java/src/jmri/server/json/operations/JsonUtil.java

```diff
@@ -328,6 +328,7 @@ public ObjectNode getRollingStock(@Nonnull RollingStock rs, Locale locale) {
         node.put(JsonOperations.LOCATION_UNKNOWN, rs.isLocationUnknown());
         if (rs.getTrack() != null) {
             node.set(JsonOperations.LOCATION, this.getRSLocationAndTrack(rs.getTrack(), rs.getRouteLocation(), locale));
+//            node.put(JsonOperations.LOCATION, rs.getLocationName() + ", " + rs.getTrackName());
```

Comment:
Just checking.
This line is added, but commented out.

Reply:
Yes, I was working on getting the location and track name displayed.  This works, but doesn't seem like the right solution.